frontend/src/services/GooglePlacesService.ts

```diff
@@ -0,0 +1,183 @@
+import axios, { AxiosError, AxiosResponse } from "axios";
+import * as dotenv from "dotenv";
+import { RestaurantDetails } from ".././types/RestaurantDetails";
+
+/* eslint-disable  no-console */
+// todo: remove this eslint-disable
+class GooglePlacesService {
+  public currRestaurants: Map<string, RestaurantDetails>;
+
+  constructor() {
+    dotenv.config();
+    this.currRestaurants = new Map();
+  }
+
+  getRestaurantPlaceID(
```

Comment:
Will this function also need to return the `lat lon` values for the backend to store?

Reply:
Nope! We call getRestaurantDetails after receiving the placeID